src/components/header.js

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import { Chromicons } from './icons/chromicons';
+import { CheckCircle, Download, Flag, Lifeology } from '@lifeomic/chromicons';
+import * as allLinedChromicons from '@lifeomic/chromicons';
+import clsx from 'clsx';
+
+const HeroNavLink = ({ className, children, ...rootProps }) => {
+  return (
+    <a
+      className={clsx(
+        'flex items-center px-2 duration-300 ease-in-out transition-opacity hover:opacity-75 focus:outline-none focus-visible:shadow-dark-mode-outline focus-visible:underline',
+        className
+      )}
+      {...rootProps}
+    >
+      {children}
+    </a>
+  );
+};
+
+export const Header = ({ pkgVersion }) => {
+  return (
+    <>
```

Comment:
Similar to how Footer returns a `<footer>` (https://github.com/lifeomic/chromicons.com/pull/57/files#diff-42bf57831737e0d5835ce4fcb459ce006fceacea526c11edd61e75fba0d7a5ccR5), Header here should be consistent and return `<header>` instead of an empty fragment.  Then you can get rid of the wrapping `<header>` tag on the index file